Jenkinsfile

```diff
@@ -46,7 +47,7 @@ pipeline {
       }
       steps {
         echo 'Deploying to Staging'
-        sh "curl --upload-file staging/target/amp-t2d-property-registry-*.war 'http://'${tomcatCredentials}'@'${stagingHost}':8080/manager/text/deploy?path=/registry&update=true' | grep 'OK - Deployed application at context path '"
+        sh "curl --upload-file staging/target/amp-t2d-property-registry-*.war 'http://'${tomcatCredentials}'@'${stagingHost}':8080/manager/text/deploy?path=/dev/registry&update=true' | grep 'OK - Deployed application at context path '"
```

Comment:
If we are already using different hosts for deployment, why is this URL also different from production? If we adopted this model all the URLs in the staging environment would be 'dev', which is redundant.

Reply:
ya i get ur point , it is because of issue with swagger and base URI , it uses context path in the base uri . we need to differentiate between https://www.ebi.ac.uk/ega/ampt2d/registry and https://www.ebi.ac.uk/ega/ampt2d/dev/registry/ . but let me check on the proxy side i could change anything once.

Reply:
its getting complex if i try to change in proxy side , may be once we update the swagger we can customize easily.